style.css

```diff
@@ -0,0 +1,503 @@
+/*global settings*/
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    outline: none;
+}
+::before,
+::after{
+ box-sizing: border-box;
+}
+:root{
+    --main-color:#e02f6b;
+    --blue: #0000ff;
+    --blue-dark:#294f7a;
+    --orange:#ffa500;
+    --green-yellow:#cddc39;
+    --pink-light:#efa2b4;
+    --cyan-light:#aef1ee;
+    --white:#ffffff;
+    --white-alpha-40:rgba(255,255,255,0.40);
+    --white-alpha-25:rgba(255,255,255,0.25);
+    --backdrop-filter-blur: blur(5px) ;
+}
+body{
+    min-height:100vh;
+    background-image: linear-gradient(to bottom right, var(--pink-light),var(--cyan-light));
+    font-size:16px;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 300;
+    color:var(--blue-dark);
+    line-height: 1.5;
+    overflow-x: hidden;
+    padding:35px 15px;
+
+}
+body::before{
+    content:'';
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+    /*background-color:var(--green-yellow) ;*/
+    z-index:-1;
+    opacity: 0.12;
+}
+a{
+    text-decoration: none;
+}
+h1,h2{
+    font-weight: 600;
+}
+h3,h4,h5,h6{
+    font-weight: 500;
+}
+img{
+    max-width: 100%;
+    vertical-align: middle;
+}
+section{
+    background-color: var(--white-alpha-25);
+    border:1px solid var(--white-alpha-25) ;
+    min-height: calc(100vh - 70px);
+    border-radius:30px;
+    backdrop-filter:var(--backdrop-filter-blur);
+    display: none;
+}
+section.active{
+    display: block;
+    animation: fadeIn 0.5s ease-in-out forwards;
+}
+section.fade-out{
+  animation: fadeOut 0.5s ease-in-out forwards;
+}
+
+button:focus {
+     outline: none;
+}
+/*background circles*/
+.bg-cirles{
+    position: fixed;
+    top: 0;
+    height: 100%;
+    max-width: 1200px;
+    width: calc(100% - 30px);
+    left: 50%;
+    transform: translate(-50%);
+}
+.bg-cirles div{
+    position: absolute;
+    border-radius: 50%;
+}
+.bg-cirles .circle-1{
+    height: 60px;
+    width: 60px;
+    background-color: var(--blue);
+    left: 5%;
+    top: 10%;
+    opacity: 35%;
+    animation: zoomInOut 8s linear infinite;
+}
+.bg-cirles .circle-2{
+    height: 100px;
+    width: 100px;
+    background-color: var(--main-color);
+    left: 35%;
+    top: 50%;
+    opacity: 40%;
+    animation: bounce 8s linear infinite;
+}
+.bg-cirles .circle-3{
+    height: 100px;
+    width: 100px;
+    background-color: var(--green-yellow);
+    right: 0%;
+    top: 70%;
+    opacity: 55%;
+    animation: bounce 8s linear infinite;
+}
+.bg-cirles .circle-4{
+    height: 45px;
+    width: 45px;
+    background-color: var(--orange);
+    left: 10%;
+    top: 65%;
+    opacity: 55%;
+    animation: zoomInOut 8s linear infinite;
+}
+.main{
+    max-width: 1200px;
+    margin: auto;
+    position: relative;
+}
+.container{
+    width: 100%;
+    padding: 0 40px;
+}
+.row{
+    display: flex;
+    flex-wrap: wrap;
+}
+.align-items-center{
+    align-items:center;
+}
+/*custom scrollbar*/
+::-webkit-scrollbar{
```

Comment:
Overriding the scrollbar properties reduces accessibility, I would recommend removing the scrollbar styles, but if you keep them, I recommend making the width bigger, as 5px is too narrow

Reply:
Okay, I'll make it bigger by increasing its width. Thank You.